cms/static/js/views/utils/move_xblock_utils.js

```diff
@@ -0,0 +1,88 @@
+            XBlockViewUtils.moveXBlock(data.sourceLocator, data.sourceParentLocator, data.targetIndex)
+            .done(function(response) {
+                // show XBlock element
+                $('.studio-xblock-wrapper[data-locator="' + response.move_source_locator + '"]').show();
+                showMovedNotification(
+                    StringUtils.interpolate(
+                        gettext('Move cancelled. "{sourceDisplayName}" has been moved back to its original ' +
+                            'location.'),
+                        {
+                            sourceDisplayName: data.sourceDisplayName
+                        }
+                    )
+                );
+            });
+        };
+
+        showMovedNotification = function(title, data) {
+            var movedAlertView;
+            if (data) {
+                movedAlertView = new AlertView.Confirmation({
+                    title: title,
+                    actions: {
+                        primary: {
+                            text: gettext('Undo move'),
+                            class: 'action-save',
+                            data: JSON.stringify({
+                                sourceDisplayName: data.sourceDisplayName,
+                                sourceLocator: data.sourceLocator,
+                                sourceParentLocator: data.sourceParentLocator,
+                                targetIndex: data.targetIndex
+                            }),
+                            click: function(view) {
+                                undoMoveXBlock(view.$el.find('.action-primary').data('primary'));
+                            }
+                        },
+                        secondary: [
+                            {
+                                text: gettext('Take me to the new location'),
+                                class: 'action-cancel',
+                                data: JSON.stringify({
+                                    targetParentLocator: data.targetParentLocator
+                                }),
+                                click: function(view) {
+                                    window.location.href = '/container/' +
+                                        view.$el.find('.action-secondary').data('secondary').targetParentLocator;
+                                }
+                            }
+                        ]
+                    }
+                });
+            } else {
+                movedAlertView = new AlertView.Confirmation({
+                    title: title
+                });
+            }
+            movedAlertView.show();
+            // scroll to top
+            $.smoothScroll({
+                offset: 0,
+                easing: 'swing',
+                speed: 1000
+            });
+            movedAlertView.$('.wrapper').first().focus();
+            return movedAlertView;
+        };
+
+        hideMovedNotification = function(SystemFeedback) {
```

Comment:
I think we should't have `SystemFeedback` as an argument, can't we define Feedback in this file and use that in this method instead of parameter?